src/gui/guiFormSpecMenu.cpp

```diff
@@ -3799,6 +3805,10 @@ bool GUIFormSpecMenu::OnEvent(const SEvent& event)
 				(kp == getKeySetting("keymap_screenshot"))) {
 			m_client->makeScreenshot();
 		}
+
+		if (event.KeyInput.PressedDown && kp == getKeySetting("keymap_toggle_debug"))
+			m_show_debug ^= true;
```

Comment:
Is there a benefit to doing an XOR instead of `m_show_debug = !m_show_debug`? I think the intent of the latter approach is more immediately clear when reading code.

Reply:
It's just the way how I usually toggle booleans (mostly in microcontrollers). There's no benefit, only personal preference. The compile might also optimize yours to an XOR operation.

Reply:
I think DF's suggestion is easier to read

Reply:
Well then, there you go.